.github/workflows/docker-image.yml

```diff
@@ -31,12 +31,22 @@ jobs:
       - name: install semver tool
         run: pip3 install semver
 
+      - name: strip v from tag
+        run: |
+          if [ ${{ github.event_name }} == 'release' ]
+          then
+            echo "Strip v from tag"
+            LAST_PART=$(echo $GITHUB_REF | cut -d / -f 3)
+            STRIPPED_VAR="${LAST_PART:1}"
+            echo "VERSION_TAG=$STRIPPED_VAR" >> $GITHUB_ENV
+          fi
+
       - name: validate release tag
         run: |
           if [ ${{ github.event_name }} == 'release' ]
           then
             echo "Validate that release tag is valid semver"
-            pysemver check $(echo $GITHUB_REF | cut -d / -f 3)
+            pysemver check $(echo $VERSION_TAG | cut -d / -f 3)
```

Comment:
VERSION_TAG already is the last part of the ref at this point. no need to do the cut (its a no op because there are no slashes in the version)